### ISSUE TITLE
Svg image show only black color 

### ISSUE THREAD

**Issue by burhan3759** [react-native-community/react-native-svg#1185]
# Bug

SVG icon show only Black color with the icon shape. 

{ SvgCss } seems missing.

## Environment info

* IOS 13
* Android 6 and above
* Basically all device have this issue

Library version: ^9.12.0

## Steps To Reproduce

svg file 

```
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="117.818" height="160" viewBox="0 0 117.818 160"><defs><style>.a{fill:url(#a);}.b{fill:url(#b);}.c{fill:#455a64;}.d{opacity:0.7;fill:url(#c);}.e{fill:url(#d);}.f{fill:#fff;}.g{opacity:0.5;fill:url(#e);}</style><linearGradient id="a" x1="-0.12" y1="0.106" x2="1.12" y2="0.894" gradientUnits="objectBoundingBox"><stop offset="0.001" stop-color="#90a4ae"/><stop offset="0.999" stop-color="#607d8b"/></linearGradient><linearGradient id="b" y1="1" x2="1" gradientUnits="objectBoundingBox"><stop offset="0" stop-color="#43d5e1"/><stop offset="1" stop-color="#32a8ed"/></linearGradient><linearGradient id="c" x1="0.26" y1="0.034" x2="1.098" y2="0.899" gradientUnits="objectBoundingBox"><stop offset="0.002" stop-color="#2389e2"/><stop offset="0.999" stop-color="#32a8ed"/></linearGradient><linearGradient id="d" x1="0.079" y1="-0.026" x2="0.921" y2="1.026" gradientUnits="objectBoundingBox"><stop offset="0.003" stop-color="#eb7070"/><stop offset="0.997" stop-color="#e54346"/></linearGradient><linearGradient id="e" x1="0.151" y1="-0.227" x2="0.952" y2="0.974" gradientUnits="objectBoundingBox"><stop offset="0.003" stop-color="#b52538"/><stop offset="0.999" stop-color="#e54346"/></linearGradient></defs><g transform="translate(-191.391 -403.757)"><rect class="a" width="101.818" height="160" rx="3" transform="translate(191.391 403.757)"/><rect class="b" width="127.516" height="80.484" transform="translate(282.543 414.423) rotate(90)"/><path class="c" d="M234.119,464.848H205.028v-7.273h29.091" transform="translate(22.727 89.697)"/><path class="d" d="M217.08,436.556l22.648,9.562h0V424.192h-16Z" transform="translate(42.815 34.058)"/><path class="e" d="M263.573,423.5,242.24,436.829,220.906,423.5v-5.333L242.24,431.5l21.333-13.333m0-5.333H220.906a5.315,5.315,0,0,0-5.333,5.333v32a5.335,5.335,0,0,0,5.333,5.333h42.667a5.334,5.334,0,0,0,5.333-5.333v-32A5.332,5.332,0,0,0,263.573,412.829Z" transform="translate(40.303 15.119)"/><path class="f" d="M260.24,420.162,238.907,433.5l-21.333-13.333v-5.333l21.333,13.333,21.333-13.333" transform="translate(43.636 18.452)"/><path class="g" d="M265.573,423.94l-5.333-7.112-21.333,13.333-21.333-13.333,18.389,32H260.24a5.332,5.332,0,0,0,5.333-5.333Z" transform="translate(43.636 21.785)"/></g></svg>
```

1. Use above code and try to display it. It will only show Black color

**Comment by msand** [comment]
Seems to work fine:

```jsx
import React from 'react';
import {SvgCss} from 'react-native-svg';
const xml = `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="117.818" height="160"
     viewBox="0 0 117.818 160">
    <defs>
        <style>
            .a{fill:url(#a);}.b{fill:url(#b);}.c{fill:#455a64;}.d{opacity:0.7;fill:url(#c);}.e{fill:url(#d);}.f{fill:#fff;}.g{opacity:0.5;fill:url(#e);}
        </style>
        <linearGradient id="a" x1="-0.12" y1="0.106" x2="1.12" y2="0.894" gradientUnits="objectBoundingBox">
            <stop offset="0.001" stop-color="#90a4ae"/>
            <stop offset="0.999" stop-color="#607d8b"/>
        </linearGradient>
        <linearGradient id="b" y1="1" x2="1" gradientUnits="objectBoundingBox">
            <stop offset="0" stop-color="#43d5e1"/>
            <stop offset="1" stop-color="#32a8ed"/>
        </linearGradient>
        <linearGradient id="c" x1="0.26" y1="0.034" x2="1.098" y2="0.899" gradientUnits="objectBoundingBox">
            <stop offset="0.002" stop-color="#2389e2"/>
            <stop offset="0.999" stop-color="#32a8ed"/>
        </linearGradient>
        <linearGradient id="d" x1="0.079" y1="-0.026" x2="0.921" y2="1.026" gradientUnits="objectBoundingBox">
            <stop offset="0.003" stop-color="#eb7070"/>
            <stop offset="0.997" stop-color="#e54346"/>
        </linearGradient>
        <linearGradient id="e" x1="0.151" y1="-0.227" x2="0.952" y2="0.974" gradientUnits="objectBoundingBox">
            <stop offset="0.003" stop-color="#b52538"/>
            <stop offset="0.999" stop-color="#e54346"/>
        </linearGradient>
    </defs>
    <g transform="translate(-191.391 -403.757)">
        <rect class="a" width="101.818" height="160" rx="3" transform="translate(191.391 403.757)"/>
        <rect class="b" width="127.516" height="80.484" transform="translate(282.543 414.423) rotate(90)"/>
        <path class="c" d="M234.119,464.848H205.028v-7.273h29.091" transform="translate(22.727 89.697)"/>
        <path class="d" d="M217.08,436.556l22.648,9.562h0V424.192h-16Z" transform="translate(42.815 34.058)"/>
        <path class="e"
              d="M263.573,423.5,242.24,436.829,220.906,423.5v-5.333L242.24,431.5l21.333-13.333m0-5.333H220.906a5.315,5.315,0,0,0-5.333,5.333v32a5.335,5.335,0,0,0,5.333,5.333h42.667a5.334,5.334,0,0,0,5.333-5.333v-32A5.332,5.332,0,0,0,263.573,412.829Z"
              transform="translate(40.303 15.119)"/>
        <path class="f" d="M260.24,420.162,238.907,433.5l-21.333-13.333v-5.333l21.333,13.333,21.333-13.333"
              transform="translate(43.636 18.452)"/>
        <path class="g"
              d="M265.573,423.94l-5.333-7.112-21.333,13.333-21.333-13.333,18.389,32H260.24a5.332,5.332,0,0,0,5.333-5.333Z"
              transform="translate(43.636 21.785)"/>
    </g>
</svg>`;
export default () => <SvgCss xml={xml} />;

```

**Comment by msand** [comment]
<img width="1680" alt="Screenshot 2019-11-08 at 5 51 15" src="https://user-images.githubusercontent.com/1131362/68450037-27d1dd80-01f2-11ea-8b6f-f5512c4169b0.png">

**Comment by burhan3759** [comment]
Hi Brother @msand, thanks for answering. I just test it with fresh install and all work's fine. Then I try with my ongoing project found out need to run **yarn upgrage react-native-svg** to fix the issue

Conclusion:
old version have some issue and need to run **yarn upgrage react-native-svg** to fix it.

**Version that have issue:-**
Library version: 9.12.0

**Latest version works fine**
Library version: 9.13.3

Thanks a lot for reply it asap.

**Comment by msand** [comment]
You're welcome! Yeah, SvgCss was published in v9.13.0 I think.
